client/src/components/Navbar/Navbar.jsx

```diff
@@ -225,37 +228,37 @@ const Navbar = () => {
 							spacing={2}
 							justifyContent={'center'}
 							alignItems={'center'}>
-							<Typography variant='h6'>
+							<Typography variant="h6">
 								Discover & read more
 							</Typography>
 							<Button
-								variant='outlined'
+								variant="outlined"
 								component={Link}
-								to='/signup'
-								size='large'
+								to="/signup"
```

Comment:
did we change any of our eslint rules? It did this automatically.

Reply:
We did not. Not sure what could have caused this